src/Pages/Search/Search.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useContext } from "react";
+import { ProductContext } from "../../Context/product-context";
+import ProductCard from "../../Components/ProductCard/ProductCard";
+import "./Search.css";
+
+export default function Search() {
+  const [search, setSearch] = useState("");
+  const { products } = useContext(ProductContext);
+  const [searchList, setSearchList] = useState([]);
+  const [successMsg, setSuccesMsg] = useState("");
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    setSearch(e.target.value);
+  };
+
+  const handleSearch = (e) => {
```

Comment:
this function is beautiful, i didn't realize when I was doing this, that we can put the category up there let alone put this whole thing in a different component, amazing thinking

Reply:
thank you